deltachat-jsonrpc/src/api/mod.rs

```diff
@@ -212,12 +212,12 @@ impl CommandApi {
         Ok(accounts)
     }
 
-    async fn start_io_for_all_accounts(&self) -> Result<()> {
+    pub async fn start_io_for_all_accounts(&self) -> Result<()> {
```

Comment:
instead of making these two methods public, you could also instantiate the CommandApi via `CommandApi::from_arc`, then you can still hold a reference to the account object yourself.

Not saying that you must/should do that, I just wanted to point out the option for now.